libvirt/helpers_test.go

```diff
@@ -293,3 +293,26 @@ func testAccCheckLibvirtNetworkLocalOnly(name string, expectLocalOnly bool) reso
 		return nil
 	}
 }
+
+// testAccCheckLibvirtNetworkDNSEnable checks the dns-enable property of the Domain
+func testAccCheckLibvirtNetworkDNSEnableOrDisable(name string, expectDNS bool) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+
+		virConn := testAccProvider.Meta().(*Client).libvirt
+
+		networkDef, err := getNetworkDef(s, name, *virConn)
+		if err != nil {
+			return err
+		}
+		if expectDNS {
```

Comment:
I think Guarde Clause is better.
Invert logic   thx

Reply:
Make sense, updated thanks.